lms/djangoapps/grades/tests/test_scores.py

```diff
@@ -47,20 +51,20 @@ class TestGetScore(TestCase):
     display_name = 'test_name'
     location = 'test_location'
 
-    SubmissionValue = namedtuple('SubmissionValue', 'exists, weighted_earned, weighted_possible')
-    CSMValue = namedtuple('CSMValue', 'exists, raw_earned, raw_possible')
+    SubmissionValue = namedtuple('SubmissionValue', 'exists, points_earned, points_possible, created_at')
```

Comment:
Why not have the Submissions API return the values as `weighted` so we can be explicit about this?  We are moving to being explicit about `raw` versus `weighted` as it tends to cause downstream confusion and errors otherwise.
Similarly, why not return `created_at` as just `created`?  We can choose the external interface of the Submissions API, instead of worrying about mimicking its internal terminology.